src/components/organisms/modals/EditableListModal/EditableListModal.tsx

```diff
@@ -257,7 +294,7 @@ const EditableListModal: FC<EditableListModalProps> = ({
         icon={Add}
         children={t('tag.add_new_row')}
         onClick={addInputField}
```

Comment:
**[blocker]**

There seems to be an issue with addInputField. Since it uses the length the the array for input names, we will start getting duplicated names, if we delete some random rows from this modal and try to add new ones after that.